packages/utils/src/promiseQueue.ts

```diff
@@ -44,12 +48,15 @@ export class PromiseQueue<T> {
 
   async run() {
     while (this.total !== this.completed) {
-      if (this.isAborted) break;
+      if (this.isAborted) return;
```

Comment:
- This will not trigger `onComplete`
- Check if `onComplete` is required if aborted

Reply:
`onComplete` is not needed if `abort` is called